src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic automatic anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+Automatic anomaly detection is the easiest way for you and your team to detect and analyze unusual behavior in your system. Automatic anomaly detection uses the following methods to detect anomalies in your app data:
+
+1. Anomaly detection monitors metric data reported by an APM agent, building a model of your typical application dynamics, and focuses on key golden signals: throughput, response time, and errors.
+2. If one of these golden signals shows anomalous behavior, the system flags the behavior and tracks recovery to normal behavior.
```

Comment:
This kind of leads me to believe that it is actively doing something to return the behavior to normal. If it doesn't, maybe it will help with a slight re-wording: "tracks recovery until behavior returns to normal"?